Library/Formula/sfml.rb

```diff
@@ -27,9 +17,41 @@ class Sfml < Formula
   depends_on "glew"
   depends_on "jpeg"
   depends_on "libsndfile"
+  depends_on "doxygen" => :optional
+
+  # Remove the bundled libs wording that causes linking errors.
+  # https://github.com/Homebrew/homebrew/pull/35279
+  patch :DATA
 
   def install
-    system "cmake", ".", "-DCMAKE_INSTALL_FRAMEWORK_PREFIX=#{frameworks}/", *std_cmake_args
+    args = std_cmake_args
```

Comment:
I'm not very familiar with brew standard variables but I guess this includes setting `CMAKE_INSTALL_PREFIX` to the the cellar, right?